starter-code/test/merge-sort-test.js

```diff
@@ -1,9 +1,31 @@
-var { expect }  = require('chai');
-var mergeSort   = require('../src/merge-sort');
+// var { expect }  = require('chai');
+// var mergeSort   = require('../src/merge-sort');
+//
+// describe('mergeSort()', function() {
+//   it('should an array of numbers sorted from least to greatest', function() {
```

Comment:
instead of `function () {` ...

the es6 way of doing it would be
```
it('should an array of numbers sorted from least to greatest', () => {
  let array = [12, 15, 0, 3, 6, 5, 10, 7, 9, 3];
  expect(mergeSort(array)).to.deep.equal([ 0, 3, 3, 5, 6, 7, 9, 10, 12, 15 ]);
});
```

notice how i changed the `function() {}` to `() => {}` and the `var` to `const`